tests/fixtures/CallbackWithDNFTypehintClass.php

```diff
@@ -7,9 +7,12 @@
 
 class CallbackWithDNFTypehintClass
 {
-    #[PHP8] public function __invoke((RuntimeException&Countable)|(RuntimeException&\IteratorAggregate) $e) { }
+    #[PHP8] public function __invoke((RuntimeException&Countable)|(RuntimeException&\IteratorAggregate) $e): void { } /*
+    public function __invoke(bool $unusedOnPhp8ButRequiredToMakePhpstanWorkOnLegacyPhp = true): void { } // */
```

Comment:
That's disgusting, but if it works until v4 I'll allow it

Reply:
Knew you were going to enjoy this! 😃 https://twitter.com/another_clue/status/1671189006162464768